src/Bicep.Core.UnitTests/Diagnostics/BicepConfigSchemaTests.cs

```diff
@@ -88,5 +89,17 @@ public void RuleConfigs_ShouldBeCorrectlyDefined()
                 refString.Should().Be("#/definitions/rule", "each rule's last allOf should be a ref to the definition of a rule");
             }
         }
+
```

Comment:
Forcing all rules to completely lowercase for consistency

Reply:
Nice!